### PR TITLE
feat(#2535): added tests and puzzles for XMIR to phi-calculus translator

### PR DESCRIPTION
**Pull request by Graur**
#2535

---

## PR-Codex overview
This PR focuses on adding functionality to the `TranslateToPhiMojo` class in the `eo-maven-plugin` module. 

### Detailed summary
- Updated the `pom.xml` file to change the maximum value from 7 to 8.
- Added a new static class `TranslateToPhi` in `FakeMaven.java` to iterate over a list of classes.
- Added a new class `TranslateToPhiMojo` in `TranslateToPhiMojo.java` to read XMIR files and translate them to phi-calculus expressions.
- Added a new test class `TranslateToPhiMojoTest` in `TranslateToPhiMojoTest.java` to test the functionality of `TranslateToPhiMojo`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by Graur** [comment]
@maxonfjvipon Please, take a look

File: eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ *  The main idea is to read program written in EO and translate it to Phi-calculus.
+ *  To store this result we suggest to create a new folder: xmir-to-phi.
+ *  Let's store it to this folder one file by one.
+ */
+@Mojo(
+    name = "xmir-to-phi",
+    defaultPhase = LifecyclePhase.PROCESS_SOURCES,
+    threadSafe = true
+)
+public final class PhiTranslatorMojo extends SafeMojo {
```
Comment (maxonfjvipon) [inline eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java:43]: @Graur I think the name of the mojo should be verb, like "parse", "optimize", "transpile"

**Review by maxonfjvipon — CHANGES_REQUESTED** [review]
@Graur good one, just one comment from my side

File: eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java (hunk 1 above)
Comment (Graur) [reply eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java:43]: @maxonfjvipon Agree. `PhiTranlate` will be ok?

Comment (maxonfjvipon) [reply eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java:43]: @Graur maybe something like "TranslateToPhi"? Or "convert"?

**Comment by Graur** [comment]
@maxonfjvipon Could you please take a look one more time?

**Review by maxonfjvipon — APPROVED** [review]
@Graur LGTM! Thanks

**Comment by Graur** [comment]
@yegor256 Please, take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**